src/Test/AbstractConfigTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractorConfig\Test;
+
+use Keboola\DbExtractorConfig\Exception\UserException;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Yaml\Yaml;
+
+abstract class AbstractConfigTest extends TestCase
+{
+    public const CONFIG_FORMAT_YAML = 'yaml';
+    public const CONFIG_FORMAT_JSON = 'json';
+
+    /** @var string */
+    protected $dataDir = __DIR__ . '/../../tests/data';
+
+    protected function getConfigDbNode(string $driver): array
+    {
+        return [
+            'user' => $this->getEnv($driver, 'DB_USER', true),
+            '#password' => $this->getEnv($driver, 'DB_PASSWORD', true),
+            'host' => $this->getEnv($driver, 'DB_HOST'),
+            'port' => $this->getEnv($driver, 'DB_PORT'),
+            'database' => $this->getEnv($driver, 'DB_DATABASE'),
+        ];
+    }
+
+    protected function getConfig(string $driver, string $format = self::CONFIG_FORMAT_YAML): array
+    {
+        switch ($format) {
+            case self::CONFIG_FORMAT_JSON:
+                $config = json_decode((string) file_get_contents($this->dataDir . '/' .$driver . '/config.json'), true);
+                break;
+            case self::CONFIG_FORMAT_YAML:
```

Comment:
Není čas už ten yaml definitivně deprekovat? @pivnicek

Reply:
Prostě updatem na verzi db-ex-common co používá tohle by se to deprekovalo..

Reply:
Tohle by mely byt DB komponenty, ktere maji nastaveny YAML:
id	name
keboola.ex-db-db2	IBM Db2
keboola.ex-db-db2-as400	DB2 (AS/400)
keboola.ex-db-db2-bata	DB2 (Bata)
keboola.ex-db-impala	Cloudera Impala
keboola.ex-db-mysql-custom	MySQL (SSL CN validation off)
keboola.ex-db-redshift-cursors	Redshift (using cursors)

Reply:
Problem s tim vyhozenim yamlu je, ze se musi soucasne zmenit verze i format konfigurace - jinak se to skarade vydrbe, coz potom dost komplikuje nejaky rollbacky a pod. Bylo by teda lepsi udelat to u dotcenejch komponent predtim, nez se budou updatovat na novy db-ex-common.

Reply:
Nicmene takhle podle voka bych rekl, ze to jsou dost okrajovy komponenty (jestli se vubec pouzivaji) a urcite bych byl pro to vyhodit z common at je to proboha pryc